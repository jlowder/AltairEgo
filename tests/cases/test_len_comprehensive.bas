10 A$ = "HELLO"
20 PRINT "LEN of 'HELLO':", LEN(A$)
30 B$ = ""
40 PRINT "LEN of empty string:", LEN(B$)
50 PRINT "LEN of literal:", LEN("WORLD")
60 C$ = "A"
70 PRINT "LEN of single char:", LEN(C$)
80 END
