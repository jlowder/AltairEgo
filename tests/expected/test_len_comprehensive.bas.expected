Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
LEN of 'HELLO':              5
LEN of empty string:         0
LEN of literal:              5
LEN of single char:          1
OK
